ci/testgrid/config.yaml

```diff
@@ -80,137 +77,104 @@ test_groups:
 - name: ci-knative-serving-performance
   gcs_prefix: knative-prow/logs/ci-knative-serving-performance
   short_text_metric: perf_latency
-- name: ci-knative-serving-auto-release
-  gcs_prefix: knative-prow/logs/ci-knative-serving-auto-release
-- name: ci-knative-serving-dot-release
-  gcs_prefix: knative-prow/logs/ci-knative-serving-dot-release
-
-# knative/build
+- name: pull-knative-serving-test-coverage
+  gcs_prefix: knative-prow/logs/ci-knative-serving-go-coverage
+  short_text_metric: coverage
 - name: ci-knative-build-continuous
   gcs_prefix: knative-prow/logs/ci-knative-build-continuous
 - name: ci-knative-build-nightly-release
   gcs_prefix: knative-prow/logs/ci-knative-build-nightly-release
-- name: ci-knative-build-auto-release
-  gcs_prefix: knative-prow/logs/ci-knative-build-auto-release
 - name: ci-knative-build-dot-release
   gcs_prefix: knative-prow/logs/ci-knative-build-dot-release
-- name: pull-knative-build-test-coverage
-  gcs_prefix: knative-prow/logs/ci-knative-build-go-coverage
-  short_text_metric: coverage
+- name: ci-knative-build-auto-release
```

Comment:
If the order of the test groups are different because of the go file, we should make them predictable. It will reduce the diff in future PRs and make reviews way easier.

Reply:
The current order is consistent with that in the template config file.